quickwit-serve/src/args.rs

```diff
@@ -19,10 +19,12 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 use std::net::SocketAddr;
+use std::path::PathBuf;
 
 #[derive(Debug)]
 pub struct ServeArgs {
     pub index_uris: Vec<String>,
     pub rest_socket_addr: SocketAddr,
-    pub peers: Vec<SocketAddr>,
+    pub host_key_path: PathBuf,
+    pub peer_socket_addr: Option<SocketAddr>,
```

Comment:
a single peer might not be a good idea.

Reply:
Changed.
Please see https://github.com/quickwit-inc/quickwit/pull/163/commits/345b8a1296897c88ded95d85444848ab0f25c484.